pkg/ast/pipesearch/wsSearchHandler.go

```diff
@@ -258,6 +258,16 @@ func processQueryUpdate(conn *websocket.Conn, qid uint64, sizeLimit uint64, scro
 		return 0
 	}
 
+	totalPossibleEvents, err := query.GetTotalRecsToBeSearchedForQid(qid)
+	if err != nil {
+		log.Errorf("qid=%d, processQueryUpdate: failed to get total records going to be searched: %+v", qid, err)
+		wErr := conn.WriteJSON(createErrorResponse(err.Error()))
+		if wErr != nil {
+			log.Errorf("qid=%d, processQueryUpdate: failed to write error response to websocket! err: %+v", qid, wErr)
+		}
+		return 0
+	}
+
```

Comment:
nit: May be create a common get function for both `totalEventsSearched` and `totalPossibleEvents` instead of calling them separately.

Reply:
addressed in the latest commit.